lib/fs/include/lib/fs.h

```diff
@@ -26,12 +26,21 @@
 #include <sys/types.h>
 #include <compiler.h>
 
-#define FS_MAX_PATH_LEN 256
-#define FS_MAX_FILE_LEN 128
+#define FS_MAX_PATH_LEN 128
+#define FS_MAX_FILE_LEN 64
```

Comment:
will have to revisit this, for the big vs little device thing. This is probably okay for now.